src/Features/CSharp/Portable/Completion/KeywordRecommenders/DelegateKeywordRecommender.cs

```diff
@@ -30,7 +30,7 @@ protected override bool IsValidContext(int position, CSharpSyntaxContext context
         {
             return
                 context.IsGlobalStatementContext ||
-                (context.IsNonAttributeExpressionContext && !context.IsConstantExpressionContext) ||
+                ((context.IsNonAttributeExpressionContext || context.IsTypeContext) && !context.IsConstantExpressionContext) ||
```

Comment:
nope.  hate it.  can't read it.  extract out into other statement with a comment.